static/bidder-info/cwire.yaml

```diff
@@ -0,0 +1,10 @@
+endpoint: "https://embed.cwi.re/prebid/ortb2"
+maintainer:
+  email: "mateja.maric@cwire.com"
+  endpointCompression: gzip

```

Comment:
I believe `endpointCompression` should be at the same level of `endpoint` and `gvlVendorID` because `maintainer` doesn't have an `endpointCompression` field.

Reply:
Fixed